src/ResendUtil.js

```diff
@@ -0,0 +1,35 @@
+import EventEmitter from 'eventemitter3'
+import { ControlLayer } from 'streamr-client-protocol'
+
+export default class ResendUtil extends EventEmitter {
+    constructor() {
+        super()
+        this.subForRequestId = {}
+        this.counter = 0
+    }
+
+    generateRequestId() {
+        const id = this.counter
+        this.counter += 1
+        return id.toString()
+    }
+
+    getSubFromResendResponse(response, responseTypeName) {
```

Comment:
It is generally expected that functions and methods with name `getXXX` do not modify state.